### DISCUSSION TITLE
Need to extract 'Ethernet' bus logging

### DISCUSSION THREAD

**Discussion by matthew-hill-tm** [danielhrisca/asammdf#617] [Q&A]
Hello,

Is there a way to extract 'Ethernet' logging from a MDF file? Currently the tool provides a way to extract both 'CAN' and 'LIN' signals but we have some 'Ethernet' signals that we would like to extract as well. Is there some plan to add this kind of feature?

**Comment by danielhrisca** [comment]
Hello Matthew,

what database format do you have to describe the ethernet bus communication?

**Reply by ecalpy** [reply]
Hi Daniel,  

Looks like this one never was followed up, but now I am facing the need to decode LIN data and I need the ability to load in a .LDF as a database (instead of .dbc or .arxml).  Is this feature being thought of already or is there a converter you can point me to to enable LIN data analysis in asammdf?

Thanks!
-Eric

**Reply by danielhrisca** [reply]
Lin decoding is already included in the latest release. You also need ldfparser and canmatrix 1.2
